test/unit/BibPage.test.js

```diff
@@ -16,9 +16,67 @@ import annotatedMarc from '../fixtures/annotatedMarc.json';
 import mockBibWithHolding from '../fixtures/mockBibWithHolding.json';
 import { makeTestStore } from '../helpers/store';
 import { mockRouterContext } from '../helpers/routing';
+import BibDetails from '../../src/app/components/BibPage/BibDetails';
+import { isAeonLink } from '../../src/app/utils/utils';
 
 describe('BibPage', () => {
   const context = mockRouterContext();
+  describe('Electronic Resources List', () => {
+    const testStore = makeTestStore({
+      bib: {
+        done: true,
+        numItems: 0,
+      },
+    });
+
+    const bib = { ...bibs[2] };
+    const page = mount(
+      <Provider store={testStore}>
+        <BibPage
+          location={{ search: 'search', pathname: '' }}
+          bib={bib}
+          dispatch={() => {}}
+          resultSelection={{
+            fromUrl: '',
+            bibId: '',
+          }}
+        />
+      </Provider>,
+      { context, childContextTypes: { router: PropTypes.object } },
+    );
+
+    it('should have an Aeon link available', () => {
+      const bttBibComp = page.findWhere(
+        (node) =>
+          node.type() === BibDetails && node.prop('additionalData').length,
+      );
+
+      expect(bttBibComp.type()).to.equal(BibDetails);
```

Comment:
```suggestion
      const bttBibComp = page.findWhere(
        (node) =>
          node.type() === BibDetails && node.prop('additionalData').length,
      );
      // The Bottom Bib Details Component has the original, Non altered, aggregated resources list.
      // It can be checked to see if the bib details would have been passed a list with Aeon links.
      
      expect(bttBibComp.type()).to.equal(BibDetails);
```